qiskit_experiments/library/__init__.py

```diff
@@ -76,6 +76,7 @@ class instance to manage parameters and pulse schedules.
     :toctree: ../stubs/
     :template: autosummary/experiment.rst
 
+    ~calibration.RoughFrequency
```

Comment:
Should we go with a naming convention to include `Cal` in cal experiment names to make it clear they are `CalibrationExperiment`s? ie `RoughFrequencyCal`